scripts/apps/monitoring/controllers/AggregateCtrl.ts

```diff
@@ -370,6 +371,26 @@ export function AggregateCtrl($scope, desks, workspaces, preferencesService, sto
         });
     }
 
+    this.isCustomFilterActive = (filter: IMonitoringFilter) => {
+        return Object.keys(this.activeFilters.customFilters ?? {}).includes(filter.label);
```

Comment:
what about simply:

```js
return this.activeFilters.customFilters?.[filter.label] != null
```

Reply:
it may not work with certain filters like `filter all items which don't have a slugline` which might be `customFilters = {slugline: null}`

Reply:
You could also use `in` notation:

```js
return filter.label in this.activeFilters.customFilters ?? {}
```

This might be more readable.

Reply:
`in` operator doesn't work the same. It also includes keys from inherited objects which not desired in most cases.

```
Object.keys(a)
[]
'a' in a
true
```

I could use a [helper function](https://user-images.githubusercontent.com/2076953/76621876-a76d1180-6530-11ea-8411-d9d24746c870.png), but it doesn't increase readability to me. What do you think?

Reply:
nah I think it's good as it is now, readable enough